tools/bin/goreleaser_utils

```diff
@@ -13,6 +13,7 @@ before_hook() {
   install_local_plugins
   install_remote_plugins
   mkdir -p "$lib_path/plugins"
+  build_standard_capabilities
```

Comment:
Do we want to always include these plugins going forward versus making it optional? If so, this makes sense.